map_service:
  voronoi_field:
    alpha: 0.001
  distance:
    default_saturation: 0.01